packages/mantine/__mocks__/@monaco-editor/react.tsx

```diff
@@ -17,8 +17,6 @@ const MockedEditor: FunctionComponent<EditorProps> = (props) => {
 
 export default MockedEditor;
 
-export const loader = {
-    config: jest.fn(),
+export const loader: any = {
```

Comment:
Can you use another type than any?

Reply:
from my understanding its a namespace, which does not seem to be usable as a type. Is there something else I can do ?

Reply:
Can you simply omit the type like it was before? TypeScript should infer it

Reply:
I get the `not portable, type annotation necessary` error from typerscript when I remove the any